test_contextual_lenses_learning.py

```diff
@@ -0,0 +1,629 @@
+# -*- coding: utf-8 -*-
+"""test_contextual_lenses_learning.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1PykmTCyuETlX4UFqoFCRpAJLIAxbAyg2
+
+# Imports
+"""
+
+import functools
+import itertools
+import os
+import time
+
+import flax
+from flax import jax_utils
+from flax import nn
+from flax import optim
+
+import jax
+from jax import random
+from jax import lax
+import jax.nn
+import jax.numpy as jnp
+
+import numpy as np
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+import scipy.stats
+
+import sklearn.linear_model
+import sklearn.metrics
+
+import seaborn as sns
+
+"""# Contextual lenses"""
+
+def max_pool(x, rep_size=None):
+  # Take maximum over sequence length
```

Comment:
Make these one-line docstrings surrounded in """ quotes and make them full sentences that end with a period.